### PR TITLE
Asvf 111

### PR DESCRIPTION
**Pull request by MrFuentastic**

### CODE REVIEW

File: app/assets/javascripts/app.js [hunk 1]
```diff
@@ -29,27 +30,32 @@ document.addEventListener("DOMContentLoaded", function(event) {
...
-          data: `code=${this.code}`,
+          data: `code=${this.codesArray}`,
           success: function(data) {
             this.results = data.results;
             this.chartData = {
               labels: data.results.map(point => point.x),
-              datasets: [
-                {
+              datasets: data.results.map((result) => {
+                var color = this.favoriteColors.pop();
```
Comment (gatorjuice) [inline app/assets/javascripts/app.js]: var is old syntax, we can use a 'let' here

File: app/assets/stylesheets/graphs.scss [hunk 2]
```diff
@@ -1,3 +1,9 @@
 // Place all the styles related to the graphs controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
```
Comment (gatorjuice) [inline app/assets/stylesheets/graphs.scss:4]: extra lines

File: public/custom.css [hunk 3]
```diff
@@ -16,6 +16,7 @@ textarea {
   top: 5%;
 }
 
+
```
Comment (gatorjuice) [inline public/custom.css]: extra line

File: test/controllers/api/v1/codes_controller_test.rb [hunk 4]
```diff
@@ -20,7 +20,7 @@ class Api::V1::CodesControllerTest < ActionDispatch::IntegrationTest
   test "Submitting actual code should return json hash containing the results of x, y coordinates" do
     post "/api/v1/code", params: {code: "[*].each do |number|\nnumber\nend"}
     data = JSON.parse(response.body)
-    assert_equal data, {"results"=>[{"x"=>100, "y"=>201}, {"x"=>500, "y"=>1001}, {"x"=>1000, "y"=>2001}, {"x"=>1500, "y"=>3001}, {"x"=>2000, "y"=>4001}, {"x"=>2500, "y"=>5001}, {"x"=>3000, "y"=>6001}]}
+    assert_equal data, {"results"=>[[{"x"=>100, "y"=>201}, {"x"=>500, "y"=>1001}, {"x"=>1000, "y"=>2001}, {"x"=>1500, "y"=>3001}, {"x"=>2000, "y"=>4001}, {"x"=>2500, "y"=>5001}, {"x"=>3000, "y"=>6001}]]}
```
Comment (gatorjuice) [inline test/controllers/api/v1/codes_controller_test.rb:23]: lets break this out so its not all one one line

**Review by gatorjuice — DISMISSED** [review]

**Review by walshification — CHANGES_REQUESTED** [review]
Let's add some test cases for processes more than one submitted code block.

**Review by walshification — APPROVED** [review]

**Pull request merged**